### ISSUE TITLE
Demand Gen and Video Campaigns

### ISSUE THREAD

**Issue by riccardomares** [jzdarsa/google_ads_script_utm_tracking#1]
Hi, your script works great to standard Google Ads campaigns,
but it seems don't work for DemandGen and Video campaigns due to the lack of the set methods for Tracking Template.

Can you confirm or did you find a workaround?

**Comment by jzdarsa** [comment]
Unfortunately Google Ads Scripts for now do not support Video and DemandGen campaigns for setting up the tracking template, this still needs to be done manually. The only option would be to start using the Google Ads API and update the campaigns settings with that.

**Comment by riccardomares** [comment]
Do you think the next API releases will solve it? 
Thanks for your reply.

**Comment by jzdarsa** [comment]
I don't think so. I think (or maybe hope) Google will come up with a different solution to add utm tracking automatically. Video is not a new campaign type and the tracking template is not supported.
